From 7b90d7908e1ec86d526b3a24b3d1c07bef0601a2 Mon Sep 17 00:00:00 2001
From: Matt Mower <mowerm@gmail.com>
Date: Wed, 28 Dec 2016 17:34:48 -0600
Subject: [PATCH 139/214] Update legal page

* Make disclaimer more professional
* Link email address
* Remove occurrences of "Android Distribution"
* Replace curly quotes with straight quotes in markdown
* Clarify language and update punctuation

Change-Id: Icb6493d1467e0370cd2a387913fb215e0e3c02ca
---
 legal.md | 30 +++++++++++++++---------------
 1 file changed, 15 insertions(+), 15 deletions(-)

diff --git a/legal.md b/legal.md
index 6f601c8..1d0e1e4 100644
--- a/legal.md
+++ b/legal.md
@@ -7,40 +7,40 @@ permalink: /legal/
 ## Terms of Service
 
 ### DISCLAIMER
-LineageOS (Lineage Android Distribution) members or anyone else on this website is not responsible for bricked devices, dead SD cards, thermonuclear war, or you getting fired because the alarm app failed. Please do some research if you have any concerns about features included in the products you find here before flashing it! YOU are choosing to make these modifications, and if you point the finger at us for messing up your device, we will laugh at you. Your warranty will be void if you tamper with any part of your device / software.
+All information and files &mdash; both in source and compiled form &mdash; are provided on an as is basis. No guarantees or warranties are given or implied. The user assumes all risks of any damages that may occur, including but not limited to loss of data, damages to hardware, or loss of business profits. Please use at your own risk. Note that unless explicitly allowed by the warranty covering your device, it should be assumed that any warranty accompanying your device will be voided if you tamper with either the system software or the hardware.
 
 ### TERMS OF USE
-Certain images, logos, pictures, and other works originating with organizations other than LineageOS may be or are the trademarks and/or servicemarks of those other organizations and are subject to the laws of their registered countries. LineageOS’ usage constitutes a ‘fair use’ of any such copyrighted material as provided for in section 107 of the US Copyright Law. If you wish to use copyrighted material from this site for purposes of your own that go beyond ‘fair use’, you must obtain permission from the copyright owner. Requests for removal of copyright material may be sent to legal@lineageos.org.
+Certain images, logos, pictures, and other works originating with organizations other than LineageOS may be or are the trademarks and/or servicemarks of those other organizations and are subject to the laws of their registered countries. LineageOS' usage constitutes a 'fair use' of any such copyrighted material as provided for in section 107 of the US Copyright Law. If you wish to use copyrighted material from this site for purposes of your own that go beyond 'fair use', you must obtain permission from the copyright owner. Requests for removal of copyright material may be sent to <legal@lineageos.org>.
 
 ## Privacy Policy
-Your privacy is important to LineageOS (Lineage Android Distribution) so we've developed a Privacy Policy that covers how we collect, use, disclose, transfer, and store your information. Please take a moment to familiarize yourself with our privacy practices.
+Your privacy is important to LineageOS. Our Privacy Policy explains how we collect, use, disclose, transfer, and store your information. Please take a moment to familiarize yourself with our privacy practices.
 
-LineageOS provides an after-market installation for various Android devices.
+LineageOS provides an after-market operating system and applications for various mobile devices.
 
 ### INFORMATION WE HAVE
-Certain information (e.g. a device unique anonymized ID, device model, etc.) is transmitted to us solely for the purpose measuring non-personally identifiable installation metrics. Unless otherwise stated below, this information is not used for any other purpose.
+Certain information (e.g. a device unique anonymized ID, device model, etc.) is transmitted to us solely for the purpose of measuring non-personally identifiable installation metrics. Unless otherwise stated below, this information is not used for any other purpose.
 
 1. Information we store from the device
- - The Unique Anonymous ID of the device. (This ID changes upon device factory reset.)
+ - The unique anonymous ID of the device (this ID changes upon device factory reset)
  - The device model
- - The version of our software installed
- - The country the device registered from
- - The carrier the device is actively registered to
+ - The version(s) of our software installed on the device
+ - The country from which the device registered
+ - The wireless carrier(s) to which the device is actively registered
 
 2. Information we store from the contributions to our code review platform (Gerrit)
  - Email addresses for registration and to facilitate review
  - Signed Contributor License Agreements (CLA)
- - IP addresses, SSH activity, and API access may be logged for rate limiting or to prevent abuse.
+ - IP addresses, SSH activity, and API access may be logged for rate limiting or to prevent abuse
 
 3. Information we store for cursory services
- - Email addresses for registration to 'wiki' and bug tracking software
- - IP addresses may be logged for rate limiting or to prevent abuse.
+ - Email addresses for registration to wiki and bug tracking software
+ - IP addresses may be logged for rate limiting or to prevent abuse
 
 4. Additional information you may provide us
  - Bug reports submitted to assist in providing support may contain Personally Identifiable Information (PII). While we will do our best to prevent the collection of this information from processes within our control, your usage and submission of this data may contain PII and you should take steps to sanitize and validate this data prior to submission.
 
 ### OPT-OUT
-You may opt-out of LineageOS device statistics via the Privacy menu on the device.
+You may opt-out of LineageOS device statistics via the Privacy menu in Settings on the device.
 
 ### WEB
 As is true of most websites, we gather some information automatically and store it in log files. This information includes Internet Protocol (IP) addresses, browser type (and version) and language, Internet service provider (ISP), referring and exit pages, operating system, date/time stamp, and clickstream data.
@@ -50,9 +50,9 @@ We may collect information regarding user activities on our website and from our
 We do not knowingly collect personal information from children under 13 except where a parent has set up an account for their child and provided LineageOS with verified parental consent. If we learn that we have collected the personal information of a child under 13 without first receiving verifiable parental consent we will take steps to delete the information as soon as possible.
 
 ### INFORMATION WE MAY SHARE
-We do not share your information with companies, organizations, and individuals outside of LineageOS unless one of the following circumstances applies:
+We do not share your information with companies, organizations, or individuals outside of LineageOS unless one of the following circumstances applies:
 
-- With your consent. (Submission of bug reports to our bug tracking platform will be considered consent.)
+- With your consent (Submission of bug reports to our bug tracking platform is considered consent.)
 - When legally required.
 
 We will share the information we have with entities outside of LineageOS if we have a good faith belief that access, use, preservation, or disclosure of the information is necessary to:
-- 
2.17.1

